src/main/java/org/sagebionetworks/bridge/spring/filters/MetricsFilter.java

```diff
@@ -38,6 +42,16 @@
     // Allow-list for query parameters metrics logging.
     private static final List<String> ALLOW_LIST =
             BridgeConfigFactory.getConfig().getList("query.param.allowlist");
+
+    /**
+     * Redis CacheProvider to give UserSession objects for Metrics.
+     */
+    CacheProvider cacheProvider;
```

Comment:
By convention, this is always private. (Same with bridgeConfig in RequestFilter.)

Reply:
Oops, I found that I actually didn't use the `cacheProvider`. So I will delete this part.